point.py

```diff
@@ -0,0 +1,16 @@
+from . import utils
+
+class Foo(object):
+    def __init__(self, x, y, mark={}):
+        self.x = x
+        self.y = y
+        self.mark = mark
```

Comment:
I wonder why you have chosen to have mark be a dictionary?  Do you image storing multiple key:value pairs?  Again, stylistic.